packages/grafana-alerting/scripts/codegen.v0alpha1.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This script will generate TypeScript type definitions and a RTKQ client for the alerting k8s APIs.
+ * It downloads the OpenAPI schema from a running Grafana instance and generates the types.
+ *
+ * Run `yarn run codegen` from the "grafana-alerting" package to invoke this script.
```

Comment:
Why not use the similar structure to [generate-rtk-apis](https://github.com/grafana/grafana/blob/552644300784d10d3a0509c3dc22b1a5fccc7b2a/scripts/generate-rtk-apis.ts) and list all the configs in one file vs a separate config per version? E.g., 
```ts
export default {
  schemaFile: '',
  apiFile: '',
  tag: true,

  outputFiles: {
    '../src/grafana/api.v0alpha1.gen.ts': {
      exportName: 'alertingAPIv0alpha1',
      /* ... */
    },
    '../src/grafana/api.v1alpha1.gen.ts': {
      exportName: 'alertingAPIv1alpha1',
      /* ... */
    },
  },
} satisfies ConfigFile;
```